pkg/cmd/codespace/create.go

```diff
@@ -34,6 +35,7 @@ func newCreateCmd(app *App) *cobra.Command {
 	createCmd.Flags().StringVarP(&opts.branch, "branch", "b", "", "repository branch")
 	createCmd.Flags().StringVarP(&opts.machine, "machine", "m", "", "hardware specifications for the VM")
 	createCmd.Flags().BoolVarP(&opts.showStatus, "status", "s", false, "show status of post-create command and dotfiles")
+	createCmd.Flags().IntVarP(&opts.idleTimeoutMinutes, "idle-timeout-minutes", "i", 30, "idle timeout in minutes")
```

Comment:
`--idle-timeout-minutes` sounds like a mouthful. Furthermore, the flags library we use support flags that are parsed into a Go "Duration" value: https://github.com/cli/cli/blob/0b8c218632fcbd24ab867ce471a1955077bd9f57/pkg/cmd/api/api.go#L220-L220

How about this being a duration flag as well? Example usage:
```
gh cs create --idle-timeout 30m
gh cs create --idle-timeout 1h
```
We could error out, of course, if the value passed by the user is less than `1m`.